src/index.js

```diff
@@ -372,4 +388,22 @@ export default class KintoClient {
     return this.execute(requests.deleteBucket(_bucket, reqOptions))
       .then(res => res.json);
   }
+
+  /**
+   * Deletes all buckets on the server.
+   *
+   * @ignore
+   * @param  {Object}  options         The options object.
+   * @param  {Boolean} options.safe    The safe option.
+   * @param  {Object}  options.headers The headers object option.
+   * @return {Promise<Object, Error>}
+   */
+  deleteBuckets(options={}) {
+    return this.ensureSupported("1.4", "2.0")
```

Comment:
No, that means we'll have to update everything to explicitly use this method for every method. And yes, I really wish we could use decorators for that ;(